### ISSUE TITLE
Several questions according to code

### ISSUE THREAD

**Issue by aezco** [Paper99/SRFBN_CVPR19#21]
Hi, I have several questions about the implementation.

1. Can I fine-tune this on the face dataset "celebA"?
2. Your results show some smoothing, although I would like to use srgan for faces to improve quality on blurry images. So for old people, I would not like to have smoothed faces, so they have wrinkles. 
3. What about training and test time?  How long does it take?

Thanks.

**Comment by Paper99** [comment]
Hi,
1. Yes, you can.
2. This is the weakness of pixel-wise losses (i.e. L2 loss and L1 loss). In your final loss function, the simplest way for generating perceptually convincing results is to add perceptual loss. You also can add the GAN loss for more photo-realistic results. 
3. It takes about 10 days to train final models. The inference time of our final model is about 0.011s on a machine with 4.2GHz Intel i7 CPU (16G RAM) and an NVIDIA 1080Ti GPU.
